src/biginteger/native.interface.js

```diff
@@ -0,0 +1,575 @@
+/* eslint-disable new-cap */
+
+/**
+ * BigInteger implementation of basic operations
+ * Wrapper of the native BigInt library
+ * Operations are not constant time
+ *  but we try and limit timing leakage where we can
+ */
+
+export default class BigInteger {
+  /**
+   * Get a BigInteger (input must be big endian for strings and arrays)
+   * @param {Number|String|Uint8Array} n value to convert
+   */
+  constructor(n) {
+    if (n instanceof Uint8Array) {
+      const bytes = n;
+      let hex = '0x0';
+      for (let i = 0; i < n.length; i++) {
+        const hexByte = bytes[i].toString(16);
+        hex += (bytes[i] <= 0xF) ? ('0' + hexByte) : hexByte;
+      }
```

Comment:
This is quite nitpicky, but when constructing long strings, it's usually faster to create an array and then at the end `.join('')` the parts together.